tuneme/urls.py

```diff
@@ -36,6 +36,7 @@
     ),
     url(r'^profiles/', include('molo.profiles.urls',
                                namespace='molo.profiles')),
+    url(r'^comments/reported/(\d+)', report_response, name='report_response'),
```

Comment:
I think you need to name this like so `url(r'^comments/reported/(?P<comment_pk>\d+)/$'`